tnp_fitter.py

```diff
@@ -259,7 +259,7 @@ def main(argv=None):
     if args.dryrun:
         print('Will run {} {} jobs'.format(len(jobs), args.command))
     elif args.condor:
-        test = False
+        test = True
```

Comment:
Don't change default arguments. If you want it to be customizable, add a flag in the argparse condor section. something like: `'test', action='store_true'`